stable/metricbeat/templates/daemonset.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: apps/v1beta2
```

Comment:
https://github.com/helm/charts "This chart repository supports the latest and previous minor versions of Kubernetes. " https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#daemonset-v1-apps states v1 and https://github.com/technosophos/k8s-helm/blob/master/docs/chart_template_guide/builtin_objects.md can be used to probe for supported api versions. this goes for the deployment api as well

Reply:
I changed to use apps/v1, thx!